alexandria/core/serializers.py

```diff
@@ -199,35 +198,27 @@ def get_webdav_url(self, instance):
         )
         return instance.get_webdav_url(username, group, host)
 
+    def validate_variant(self, value):
+        if not value:
+            value = "original"
```

Comment:
Isn't it required anyways?

Reply:
simplified in other PR: `return "original"`